### PR TITLE
feat: lookup stealth meta address

### PR DESCRIPTION
**Pull request by marcomariscal**

### CODE REVIEW

File: examples/getStealthMetaAddress/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Get Stealth Meta Adress Example
```
Comment (garyghayrat) [inline examples/getStealthMetaAddress/README.md:1]:
```suggestion
# Get Stealth Meta Address Example
```

File: examples/getStealthMetaAddress/index.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import {
+  createStealthClient,
+  getStealthMetaAddress,
+  VALID_SCHEME_ID,
+  ERC6538_CONTRACT,
+} from 'stealth-address-sdk';
+
+// Example stealth client parameters
+const chainId = 11155111; // Example chain ID for Sepolia
+const rpcUrl = 'https://sepolia.infura.io/v3/YOUR_INFURA_PROJECT_ID';
```
Comment (garyghayrat) [inline examples/getStealthMetaAddress/index.ts:10]: This key should be taken out into a `.env.exmaple`

File: src/lib/abi/ERC6538Registry.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        type: 'uint256',
+      },
+      {
+        indexed: false,
+        internalType: 'bytes',
+        name: 'stealthMetaAddress',
+        type: 'bytes',
+      },
+    ],
+    name: 'StealthMetaAddressSet',
```
Comment (garyghayrat) [inline src/lib/abi/ERC6538Registry.ts:26]: Just fyi, we have a new contract with an additional event coming soon.

File: src/lib/abi/index.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export { default as ERC5564RegistryAbi } from './ERC5564Announcer';
```
Comment (garyghayrat) [inline src/lib/abi/index.ts:1]:
```suggestion
export { default as ERC5564AnnouncerAbi } from './ERC5564Announcer';
```

File: examples/getStealthMetaAddress/index.ts (hunk 2 above)
Comment (marcomariscal) [reply examples/getStealthMetaAddress/index.ts]: included [here](https://github.com/ScopeLift/stealth-address-sdk/pull/19/commits/00391af1679c789884a727a5dbb6b8274489c9cf)

**Review by garyghayrat — APPROVED** [review]
lgtm! Thank you!

**Pull request merged**